usage_examples/el_openai/run_pipeline.py

```diff
@@ -0,0 +1,57 @@
+import os
+from pathlib import Path
+from typing import Optional
+
+import spacy.util
+import typer
+from spacy.tokens import Doc
+from wasabi import msg
+
+from spacy_llm.tests.tasks.test_entity_linker import build_el_pipeline
```

Comment:
This usage example shouldn't be using code from the test suite. I suggest to copy this function in here, so users can actually see what is happening directly.

Reply:
And even more ideally, we make this less convoluted, and provide a way for users to "just" define the `el_kb_data.yml` and everything else happens behind the scenes?

Reply:
Suggestion for this in https://github.com/explosion/spacy-llm/pull/241/commits/3271daf915c96d1f576888ee2a4d5b6d484606d1 and https://github.com/explosion/spacy-llm/pull/241/commits/94c22e8d295981ead3f868aac7a19046adac42d5.

Reply:
Just noticed that the description file is redundant if a KB .yaml file is provided, so this can be further simplified. The current state is good enough for a review though.

Reply:
Sounds good! Feel free to assign this back to me for review when it is further simplified, the registry functions done and the usage example here cleaned up.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/785dd3b51f22a3c7a64f6e412491c1203a088016.